conversions/prefix_conversions.py

```diff
@@ -0,0 +1,111 @@
+"""
+Convert SI and Binary prefixes
```

Comment:
Does SI stand for System Integrator or Serial Interface and why does the reader have to guess?

Please add the URL https://en.wikipedia.org/wiki/International_System_of_Units or similar to help the reader.